src/cert_store.rs

```diff
@@ -150,6 +171,46 @@ impl CertStore {
         }
     }
 
+    /// Deletes a specified store within the context of the current user.
+    /// 
+    /// Deleting stores is not natural and should only be done if necessary
+    pub fn delete_current_user_store(which: &str) -> () {
+        unsafe {
+            let data = OsStr::new(which)
+                .encode_wide()
+                .chain(Some(0))
+                .collect::<Vec<_>>();
+            let _store = Cryptography::CertOpenStore(
```

Comment:
>_store [](http://example.com/codeflow?start=16&length=6)

Add comment, that NULL is always returned when the CERT_STORE_DELETE_FLAG is set. Need to check last error for success.